actions_test.go

```diff
@@ -328,3 +333,16 @@ func TestAction_WithFieldsMap(t *testing.T) {
 		t.Errorf("expected %q to be %q", got, want)
 	}
 }
+
+// newFakeGetenvFunc returns a new getenvFunc that is expected to be called with
+// the provided key. It returns the provided value if the call matches the
+// provided key. It reports an error on test t otherwise.
+func newFakeGetenvFunc(t *testing.T, wantKey, v string) getenvFunc {
```

Comment:
Nit: you generally want to accept `testing.TB` instead of `*testing.T` so that the interface works with both benchmarks and tests.